src/main/java/io/confluent/connect/jdbc/JdbcUtils.java

```diff
@@ -154,5 +155,17 @@ public static String getAutoincrementColumn(Connection conn, String table) throw
   public static String formatUTC(Date date) {
     return DATE_FORMATTER.get().format(date);
   }
+
+  /**
+   * Get the string used for quoting identifiers in this database's SQL dialect.
+   * @param connection the database connection
+   * @return the quote string
+   * @throws SQLException
+   */
+  public static String getIdentifierQuoteString(Connection connection) throws SQLException {
+    String quoteString = connection.getMetaData().getIdentifierQuoteString();
+    quoteString = quoteString == null ? "" : quoteString;
```

Comment:
Haha, as if you can trust it to be implemented properly in the first place. Some JDBC drivers apparently return null.